.github/workflows/publish.yml

```diff
@@ -0,0 +1,76 @@
+name: Publish web package to npmjs
+
+# This workflow runs when code is pushed to `main` (i.e: when a pull request is merged)
+on:
+    push:
+        branches: [main]
+
+jobs:
+    version:
+        runs-on: ubuntu-latest
+        steps:
+            # Running this action ensures that only one instance of this job will run at a time.
+            # This is important to prevent race conditions when multiple pull requests are merged in quick succession
+            - uses: softprops/turnstyle@8db075d65b19bf94e6e8687b504db69938dc3c65
+              with:
+                  poll-interval-seconds: 10
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+            - uses: actions/checkout@v3
+
+            - uses: actions/setup-node@v3
+              with:
+                node-version: '16.x'
+                registry-url: 'https://registry.npmjs.org'
+
+            - name: Generate branch name
+              run: echo "BRANCH_NAME=botify-bump-version-$(uuidgen)" >> $GITHUB_ENV
+
+            - name: Create branch for version-bump pull request
+              run: git checkout -b ${{ env.BRANCH_NAME }}
+
+            - name: Install npm packages
+              run: npm ci
+
+            - name: Update npm version
+              run: npm version patch
+
+            - name: Set new version in GitHub ENV
+              run: echo "NEW_VERSION=$(npm info react-native-onyx version)" >> $GITHUB_ENV
+
+            - name: Build web code
```

Comment:
Oh, now the webpack configs are all exported from the same file, so it only needs one build command. Let's just rename it from `build:web` to `build` since it's no longer web specific.

Reply:
👍